frontend/app/components/input-field.tsx

```diff
@@ -35,6 +36,7 @@ const InputField = forwardRef<HTMLInputElement, InputFieldProps>((props, ref) =>
       <InputLabel id={inputLabelId} htmlFor={id} required={required} className="mb-2">
         {label}
       </InputLabel>
+      {note && <p className="mb-2">{note}</p>}
```

Comment:
For consistency with the `<InputRadios />` component, I suggest adding primary and secondary help properties instead of `note`.

See https://github.com/DTS-STN/canadian-dental-care-plan/pull/545/files

Reply:
fixed